pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -297,7 +297,7 @@ func (r *ReconcileComplianceScan) phasePendingHandler(instance *compv1alpha1.Com
 	if instance.NeedsTimeoutRescan() {
 		instanceCopy := instance.DeepCopy()
 		delete(instanceCopy.Annotations, compv1alpha1.ComplianceScanTimeoutAnnotation)
-		delete(instanceCopy.Annotations, compv1alpha1.ComplianceScanTimeoutAnnotation)
+		delete(instanceCopy.Annotations, compv1alpha1.ComplianceCheckCountAnnotation)
```

Comment:
Interesting - looks like we added this in f52006b3a

I wonder if we can consolidate this conditional with the one above. They're pretty similar.

Reply:
Picking this up in https://github.com/ComplianceAsCode/compliance-operator/pull/693